src/api/types.ts

```diff
@@ -68,6 +68,9 @@ export interface OHRIFormSchema {
   inlineRendering?: 'single-line' | 'multiline' | 'automatic';
   markdown?: any;
   postSubmissionActions?: Array<string>;
+  formOptions?: {
+    usePreviousValueDisabled: boolean;
+  };
```

Comment:
We may need to move other options like:
- inlineRendering
- readonly
- defaultPage
-  allowUnspecifiedAll
- postSubmissionActions

Under `formOptions`. @kajambiya Can you log a ticket for this?

Reply:
Sure. Let me do that